### PR TITLE
Update localization sample and make it cli-ready

### PR DESCRIPTION
**Pull request by pakrym**
@DamianEdwards @davidfowl

### CODE REVIEW

File: samples/LocalizationSample/Startup.cs [hunk 1]
```diff
@@ -49,7 +52,7 @@ public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
                     new CultureInfo("fr-FR"),
                     new CultureInfo("zh"),
                     new CultureInfo("zh-CN"),
-                    new CultureInfo("zh-CHT")
+                    //new CultureInfo("zh-CHT")
```
Comment (pakrym) [inline samples/LocalizationSample/Startup.cs]: CoreCLR doen't support this.

```
Globalization.CultureNotFoundException: Culture is not supported.
Parameter name: name
zh-CHT is an invalid culture identifier.
   at System.Globalization.CultureInfo..ctor(String name, Boolean useUserOverride)
```

Comment (davidfowl) [inline samples/LocalizationSample/Startup.cs]: Ifdef?

Comment (pakrym) [inline samples/LocalizationSample/Startup.cs]: ok

**Comment by pakrym** [comment]
:up: :date:

File: samples/LocalizationSample/Startup.cs (hunk 1 above)
Comment (DamianEdwards) [inline samples/LocalizationSample/Startup.cs]: This surprises me, given that full framework on Windows 10 will let anything through. Have you tried on non-Windows?

Comment (pakrym) [inline samples/LocalizationSample/Startup.cs]: I tried on Windows 10

Comment (DamianEdwards) [inline samples/LocalizationSample/Startup.cs]: :confused:

**Comment by DamianEdwards** [comment]
:shipit:

**Comment by hishamco** [comment]
This PR should react to the new configurationOptions lambda

**Comment by hishamco** [comment]
@pakrym i think you need to react changes to the commit d9fc39953143367a2fc3dab650859dc1a06b328e

File: samples/LocalizationSample/Startup.cs [hunk 2]
```diff
@@ -16,6 +17,8 @@ namespace LocalizationSample
 {
     public class Startup
     {
+        public static void Main(string[] args) => WebApplication.Run<Startup>(args);
```
Comment (davidfowl) [inline samples/LocalizationSample/Startup.cs]: Wut?

Comment (pakrym) [inline samples/LocalizationSample/Startup.cs]: It was long ago ) Fixing..

**Comment by pakrym** [comment]
:up: :date:

**Comment by Eilon** [comment]
@pakrym are you just waiting for further code review?

**Comment by pakrym** [comment]
@Eilon, I had issues testing using cli on ubuntu and then forgot it completely forgot about it. Will retest/merge tomorrow.

**Comment by Eilon** [comment]
Thanks!

**Comment by Eilon** [comment]
@pakrym ping again :smile:

**Pull request merged**